src/sim-host/ui/custom-elements.js

```diff
@@ -412,6 +480,14 @@ function registerCustomElement(name) {
     });
 }
 
+function getId() {
+    var result = [];
+    for (var i = 0; i < 8; i++) {
+        result.push((Math.random() * 16 | 0).toString(16));
```

Comment:
I also believe that IDs must start with a letter and not a number, which means this can generate some invalid identifiers since we use it as a prefix